### DISCUSSION TITLE
Trouble installing on Ubuntu. Cannot run as sudo

### DISCUSSION THREAD

**Discussion by vince-p** [Byron/dua-cli#166] [Q&A] [answered]
I'm running a clean install of ubuntu server.

I used the following commands to install dua:
`curl -LSfs https://raw.githubusercontent.com/byron/dua-cli/master/ci/install.sh | \
    sh -s -- --git byron/dua-cli --target x86_64-unknown-linux-musl --crate dua --tag v2.20.1`

That ran successfully but the command 'dua' did not work, I got a "command not found error".
I added ~/.cargo/bin to my PATH, and now it does run.

But I cannot do 'sudo dua'. I again get 'command not found'.

Is there a way to install on Ubuntu so that I dont have to worry about paths and can run the program as sudo?
Alternately, how can run the command as sudo in my current situation.
Thanks

**Comment by vince-p** [comment]
I also tried deleting everything, installing rustup and installing through cargo.
This results in the same situation: dua runs as my user but when I try 'sudo dua' i get 'command not found'

**Comment by Byron** [comment, marked as answer]
I think this is a problem unrelated to `dua` as it truly has nothing to do with `PATH` or `sudo`, nor does it do anything to deal with its own privileges. This question might find an answer in more general terms on stackoverflow for example. Maybe [this helps as well](https://chat.openai.com/share/d2eb0f53-f13e-4cd9-b257-b49d802736fc).

The following does the trick, validated on ubuntu.

```sh
sudo `which dua`
```

**Reply by vince-p** [reply]
Agree this is not a problem with the dua program and this is a totally fair response.
However it would be cool to provide a method or instructions to get dua fully working on Ubuntu.

**Reply by Byron** [reply]
Folks finding this discussion will now be presented with the exact command-line to invoke.
